src/jvmTest/kotlin/de/voize/semver4k/SemverTest.kt

```diff
@@ -361,6 +361,54 @@ class SemverTest {
         Assert.assertFalse(Semver("0.1.2").isStable)
     }
 
+    @Test
+    fun findsNextIncrements() {
+        Assert.assertEquals(Semver("1.0.1"), Semver("1.0.0").nextIncrement())
+        Assert.assertEquals(Semver("1.0.0-beta.0"), Semver("1.0.0-beta").nextIncrement())
+        Assert.assertEquals(Semver("1.0.0-beta.5"), Semver("1.0.0-beta.4").nextIncrement())
+        Assert.assertEquals(Semver("1.0.0-beta.5+sha899-d8g79f87"), Semver("1.0.0-beta.4+sha899-d8g79f87").nextIncrement())
```

Comment:
can you also add a test for pre 1.0.0 version

Reply:
0.1.0 for example

Reply:
should be good now